app/namespace.go

```diff
@@ -335,7 +335,19 @@ func (c *NamespaceClient) deleteRegion(ctx *cli.Context) error {
 	return PrintProto(res.GetAsyncOperation())
 }
 
-func (c *NamespaceClient) listNamespaces() error {
+func (c *NamespaceClient) listNamespaces(requestedPageToken string, pageSize int) error {
+	if len(requestedPageToken) > 0 || pageSize > 0 {
```

Comment:
nit: do we want to upper bound check the pageSize so that it matches the API so it won't make the API call if it's too many?

Reply:
yeah, let me add a check.